evaluate_library_access_from_output_tsv/stratefied_sample_of_dois_and_manual_full_text_check_results.tsv

```diff
@@ -0,0 +1,201 @@
+"doi"	"oadoi_color"	"full_text_indicator_automated"	"date_of_manual_full_text_check"	"full_text_indicator_manual"
```

Comment:
Quotes not necessary.

In general, your file names are way too long and verbose. Clarity is good, but long names become difficult to work with... throwing out some ideas: `accuracy-analysis.tsv`, `manual-dois.tsv`...

As an example, the directory name here is `evaluate_library_access_from_output_tsv`. Is `_from_output_tsv` necessary documentation to include at the directory naming level?

Reply:
> Quotes not necessary.

To confirm, following your [comment](https://github.com/greenelab/library-access/pull/17#discussion_r156502860) to use `reader::write_tsv`, quotes will no longer be written by default.

> throwing out some ideas: `accuracy-analysis.tsv`, `manual-dois.tsv`

As of e22a83cb9c, I have renamed the `tsv` file to `manual-doi-checks.tsv`, following your suggestion. To you, does that meet the spirit of the suggestion that you wrote? If not, would you prefer that i name the file `manual-dois.tsv` exactly?

> Is `_from_output_tsv` necessary documentation to include at the directory naming level?

[Shrug] This seems like an aesthetic choice to me -- I do find more verbose filenames useful in my daily work. If you're worried that the verbosity will be distracting to others viewing this code in the future, or if it's distracting to you now, I don't feel invested in keeping it the way it is. As the project lead, is there a name to which you would prefer I change this directory name, as well?

Reply:
`manual-doi-checks.tsv` is great.

> This seems like an aesthetic choice to me

Indeed. The issue is that it takes longer to read longer names and that URLs / file paths become unwieldy. Of course, it's a balance and overly long names are better than overly short names. Something like `manual-doi-checks` could be a good directory name... the idea is that everything related to the manual DOI checks will be in this directory. It captures the scope of the contents well?

Reply:
(For example, `evaluate_library_access` seems ambiguous to me, since we're specifically evaluating the automated query's report on access. But `evaluate_automated_access_output`, e.g., is not much shorter than `evaluate_library_access_from_output_tsv`.

I'm writing this since you asked whether the extra characters were necessary documentation at the naming level, though I suspect you meant it as a rhetorical question. I appreciate your note, and take your point; if you did mean that as a rhetorical question, as you're project lead, I'm willing to just follow / implement naming conventions/requirements that you have, if there's something you'd like the directory changed to.)